integration_tests/test_checkov_cli_integration_report.py

```diff
@@ -8,6 +9,15 @@
 
 class TestCheckovJsonReport(unittest.TestCase):
 
+    def test_guideline_in_custom_policy(self):
```

Comment:
@mikeurbanski1 This test should run only when we have the API key, which is embodied in this:
```python
if sys.version_info[1] == 7 and platform.system() == 'Linux':
```